assets/js/state/sagas/index.js

```diff
@@ -205,6 +208,10 @@ function* watchHeartbeatFailed() {
   yield takeEvery('HEARTBEAT_FAILED', heartbeatFailed);
 }
 
+function* watchHostDeregistered() {
+  yield takeEvery('HOST_DEREGISTERED', hostDeregistered);
```

Comment:
Let's move this `watch` to the saga files (as we do with the catalog and lastExecutions).
Besides that, `HOST_DEREGISTERED` should be a constant.
By now, these are stored in the `actions` file, but with the latest ADR i'm not sure where this should go. Reducer or saga? @dottorblaster wdyt?